src/core/config/commissioner.h

```diff
@@ -55,4 +55,15 @@
 #define OPENTHREAD_CONFIG_COMMISSIONER_MAX_JOINER_ENTRIES 2
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_COMMISSIONER_JOINER_SESSION_TIMEOUT
+ *
+ * The timeout for the Joiner's session. After this timeout, the Commissioner
+ * tears down the session.
```

Comment:
This comment should mention that `OPENTHREAD_CONFIG_COMMISSIONER_JOINER_SESSION_TIMEOUT` is in seconds.